_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -64,185 +65,301 @@ curl -X PUT "$host/_cluster/settings?pretty" -H 'Content-Type: application/json'
+- 4 primary shards and 4 replica shards per node
 
 <table>
+    <th colspan="2" ></th>
+    <th colspan="3" >40 GB primary shard, 80 GB total</th>
+    <th colspan="3">240 GB primary shard, 480 GB total</th>
     <tr>
         <td></td>
         <td></td>
-        <td><b>Document Replication</b></td>
-        <td><b>Segment Replication</b></td>
-        <td><b>Percent difference</b></td>
+        <td>Document Replication</td>
+        <td>Segment Replication</td>
+        <td>Percent difference</td>
+        <td>Document Replication</td>
+        <td>Segment Replication</td>
+        <td>Percent difference</td>
     </tr>
     <tr>
-        <td><b>Test execution time (minutes)</b></td>
-        <td></td>
-        <td>118.00</td>
-        <td>98.00</td>
-        <td>27%</td>
+        <td>Store size</td>
+        <td ></td>
+        <td>85.2781</td>
+        <td>91.2268</td>
+        <td>N/A</td>
+        <td>515.726</td>
+        <td>558.039</td>
+        <td>N/A</td>
     </tr>
     <tr>
-        <td rowspan="3"><b>Index Throughput (number of requests per second)</b></td>
-        <td>p0</td>
-        <td>71917.20</td>
-        <td>105740.70</td>
-        <td>47.03%</td>
+        <td rowspan="3">Index throughput (number of requests per second)</td>
+        <td>Minimum</td>
+        <td>148,134</td>
+        <td>185,092</td>
+        <td>24.95%</td>
+        <td>100,140</td>
+        <td>168,335</td>
+        <td>68.10%</td>
     </tr>
     <tr>
-        <td>p50</td>
-        <td>77392.90</td>
-        <td>110988.70</td>
-        <td>43.41%</td>
+        <td class="td-custom">Median</td>
+        <td>160,110</td>
+        <td>189,799</td>
+        <td>18.54%</td>
+        <td>106,642</td>
+        <td>170,573</td>
+        <td>59.95%</td>
     </tr>
     <tr>
-        <td>p100</td>
-        <td>93032.90</td>
-        <td>123131.50</td>
-        <td>32.35%</td>
-    </tr>
-     <tr>
-        <td rowspan="3"><b>Query Throughput (number of requests per second)</b></td>
-        <td>p0</td>
-        <td>1.748</td>
-        <td>1.744</td>
-        <td>-0.23%</td>
+        <td class="td-custom">Maximum</td>
+        <td>175,196</td>
+        <td>190,757</td>
+        <td>8.88%</td>
+        <td>108,583</td>
+        <td>172,507</td>
+        <td>58.87%</td>
     </tr>
     <tr>
-        <td>p50</td>
-        <td>1.754</td>
-        <td>1.753</td>
-        <td>0%</td>
+        <td>Error rate</td>
+        <td ></td>
+        <td>0.00%</td>
+        <td>0.00%</td>
+        <td >0.00%</td>
+        <td>0.00%</td>
+        <td>0.00%</td>
+        <td>0.00%</td>
     </tr>
+</table>
+
+As the size of the workload increases, the benefits of segment replication are amplified because the replicas are not required to index the larger dataset. In general, segment replication leads to a higher throughput at a lower resource cost than document replication in all cluster configurations, not accounting for replication lag. 
```

Comment:
The end of the first sentence reads as though the replicas are taking the action. Is that what we mean, or do we mean "the replicas are not required in order to index the larger dataset"? In the second sentence, we could probably just say "leads to higher throughput at lower resource cost".

Reply:
Yes, in the first sentence we do mean the replicas do not have to take action.